humans_nav/HumanPose[]       goals
---
bool                         success
string                       message
